include/node.hpp

```diff
@@ -473,6 +473,31 @@ struct node : protected std::tuple<Arguments...> {
     init(std::shared_ptr<gr::Sequence> progress_, std::shared_ptr<fair::thread_pool::BasicThreadPool> ioThreadPool_) {
         progress     = std::move(progress_);
         ioThreadPool = std::move(ioThreadPool_);
+
+        // Set names of port member variables
+        auto setPortName = [&]([[maybe_unused]] std::size_t index, auto &&t) {
```

Comment:
Why do we need this? aren't the reflected names enough?

Reply:
The port instance does not know its variable name. This is a bridge between the compile-time refl-name and the runtime name of the port.

Reply:
There is already API for that in the Port. Either we need to use 
 * statically 'named' ports (NTTP Port API), 
 * names that are derived from their field names, or
 * names that are derived from their field names and their index. Here the implicit assumption is that e.g. the 2 port in an array of ports would be called "in1". This minimises ambiguity and crafting unconventional names.